sfml/squash_game/Game.h

```diff
@@ -0,0 +1,25 @@
+#ifndef GAME_H
```

Comment:
your example doesn't start with __. 
Also, from what I've heard, "According to the C++ Standard, identifiers starting with one underscore are reserved for libraries. Identifiers starting with two underscores are reserved for compiler vendors."(http://stackoverflow.com/questions/224397/why-do-people-use-double-underscore-so-much-in-c)